source/core/src/main/com/csse3200/game/components/combat/CombatManager.java

```diff
@@ -273,14 +292,19 @@ private Entity getFasterEntity() {
     private void checkCombatEnd() {
         if (playerStats.getHealth() <= 0) {
             if (enemy.getComponent(CombatStatsComponent.class).isBoss()) {
+                //nullifyCombatDialogueListener();
                 this.getEntity().getEvents().trigger("combatLossBoss");
                 GameState.resetState();
                 SaveHandler.delete(GameState.class, "saves", FileLoader.Location.LOCAL);
             } else {
                 this.getEntity().getEvents().trigger("combatLoss");
+                //nullifyCombatDialogueListener();
```

Comment:
bunch of commented out lines in this file. do u need them or can they be deleted?